lib/streamlit/connections/sql_connection.py

```diff
@@ -44,14 +44,38 @@
 
 
 class SQL(ExperimentalBaseConnection["Engine"]):
-    """A thin wrapper around SQLALchemy that makes it play nicely with
-    st.experimental_connection.
+    """A connection to a SQL database using a SQLAlchemy Engine.
+    Initialize using ``st.experimental_connection("<name>", type="sql")``.
 
-    The SQL connection also provides the `query` convenience method, which can be used to
+    The SQL connection provides the ``query()`` convenience method, which can be used to
```

Comment:
@sfc-gh-jcarroll Johannes had requested adding callouts to the effect of "don’t create this class directly but use st.connection etc" to some of these classes. Let's add a sentence to all such places?

Reply:
IMO it's covered by the line above 

> Initialize using ``st.experimental_connection("<name>", type="sql")``.

And we have the same thing on Snowpark.

@vdonato what do you think? Could make it _even_ more explicit but not sure it's needed?

Reply:
I think it'd be useful to be extra explicit about calling this out. My initial thought was that this could be mentioned as an additional point in the documentation site but not in the docstring, but I think the current doc prototypes mostly render the docstrings nicely, so it'd be good to just include this here

Reply:
How's the update look?